docs/src/test/java/docs/http/javadsl/server/JsonStreamingExamplesTest.java

```diff
@@ -84,24 +80,31 @@ final Route tweets() {
       )
     );
     //#response-streaming
+    return responseStreaming;
+  }
+
+  final Route measurements() {
```

Comment:
Split the `Route` into two, as he incoming streaming example should be for `Measurement` not `Tweet`, which is described in the "Consuming JSON Streaming uploads" section.